pwiz_tools/Skyline/TestTutorial/SmallMoleculesMethodDevAndCEOptimizationTutorial.cs

```diff
@@ -257,23 +266,32 @@ protected override void DoTest()
                     Assert.IsTrue(string.IsNullOrEmpty(msg), msg);
                 RestoreViewOnScreen(9);
                 SelectNode(SrmDocument.Level.MoleculeGroups, 0);
+                WaitForGraphs();
+                RunUI(() =>
+                {
+                    AdjustSequenceTreePanelWidth();
+                    var ch = SkylineWindow.GraphChromatograms.FirstOrDefault((chrom) => chrom.NameSet == "unscheduled_2min");
+                    ch?.ZoomTo(.85, 2.1, 1.9e+8);
+                });
+
                 PauseForScreenShot<SkylineWindow>("Skyline window multi-replicate layout", 10);
 
-                // Set zoom to show better peak seperation in 5 minute run
+                // Set zoom to show better peak separation in 5 minute run
                 for (var index = 0; index < 2; index++)
                 {
                     var indexChrom = index;
                     WaitForGraphs();
-                    RunUI(() => SkylineWindow.GraphChromatograms.ToArray()[indexChrom].ZoomTo(.8, 1.8));
+                    RunUI(() => SkylineWindow.GraphChromatograms.ToArray()[indexChrom].ZoomTo(.8, 1.8, 1.39e+8));
                     WaitForGraphs();
                 }
-                PauseForScreenShot<SkylineWindow>("Skyline window showing relative peak seperation", 12);
+                PauseForScreenShot<SkylineWindow>("Skyline window showing relative peak separation", 12);
 
                 // Set time window
                 var peptideSettingsDlg = ShowDialog<PeptideSettingsUI>(SkylineWindow.ShowPeptideSettingsUI);
                 RunUI(() =>
                 {
-                    peptideSettingsDlg.SelectedTab = PeptideSettingsUI.TABS.Prediction;
+                    // ReSharper disable once RedundantCast
+                    peptideSettingsDlg.SelectedTab = (PeptideSettingsUI.TABS)0; //regular enum does not work because of the hidden tabs in the Small Molecule mode.

```

Comment:
We should fix the implementation of PeptideSettingsUI.SelectedTab so that it works correctly in terms of returning valid values from the enum PeptideSettingsUI.TABS.

Probably, we would make it so that "SelectedTab" has a switch statement in it, and maps between TabPage objects such as "tabQuantification" and enum values TABS.Quantification